rest/blip_sync.go

```diff
@@ -231,8 +231,8 @@ func (bh *blipHandler) handleSetCheckpoint(rq *blip.Message) error {
 // Received a "subChanges" subscription request
 func (bh *blipHandler) handleSubscribeToChanges(rq *blip.Message) error {
 
-	subChanges := newSubChanges(rq, bh.blipSyncContext, bh.db.CreateZeroSinceValue(), bh.db.ParseSequenceID)
-	bh.logEndpointEntry(rq.Profile(), subChanges)
+	subChanges := newSubChangesParams(rq, bh.blipSyncContext, bh.db.CreateZeroSinceValue(), bh.db.ParseSequenceID)
```

Comment:
I feel like changing the local variable from subChanges to subChangesParams would actually be a big improvement in readability.   It makes subsequent usage (like subChanges.since()) much more self-documenting.  (i.e. I don't have to work out whether subChanges.since() refers to the current (mutable) since value in a continuous replication, or if it's just the request parameter).

Not critical to change if you think it makes the code too verbose.

Reply:
Yeah, I missed that and will update.